### PR TITLE
refactor: extract fees logic from task into reusable functions

### PR DESCRIPTION
**Pull request by fadeev**
Making it easier to reuse these functions in the future in the browser environment.

### CODE REVIEW

**Comment by fadeev** [comment]
Withdraw fee is already the total fee, so we don't need to manually add to get total.

https://github.com/zeta-chain/protocol-contracts/blob/038b28c5679b0da145e1822f4b16df9aa30dc2bd/contracts/zevm/ZRC20.sol#L268

I am substracting the protocol fee from the withdraw fee to get the gas part for consistency.

File: helpers/fees.ts [hunk 1]
```diff
@@ -28,46 +18,37 @@ export const getFee = async (
...
-  const contract = new hre.ethers.Contract(zrc20Address, ZRC20.abi, provider);
-
-  const gasFee = hre.ethers.BigNumber.from(
-    (await contract.withdrawGasFee())[1]
-  );
-  const protocolFee = hre.ethers.BigNumber.from(
-    await contract.PROTOCOL_FLAT_FEE()
-  );
+  const contract = new ethers.Contract(zrc20Address, ZRC20.abi, provider);
+  const gasFee = ethers.BigNumber.from((await contract.withdrawGasFee())[1]);
```
Comment (andresaiello) [inline helpers/fees.ts:39]: can you split this in two lines with some aux variable? is confusing the await(await()[index])

Comment (fadeev) [reply helpers/fees.ts]: https://github.com/zeta-chain/toolkit/pull/30/commits/7509a24e1ca797f1d3445f619bdd16d2b098037a

**Comment by fadeev** [comment]
I've also removed `getFees` as it was dispatching one of the two function based on an argument, kind of useless.

**Review by andresaiello — APPROVED** [review]

**Pull request merged**